lib/pbench/server/cache_manager.py

```diff
@@ -771,35 +772,26 @@ def extract(tarball_path: Path, path: str) -> Inventory:
             tarball_path, f"Unexpected error from {tar_path}: {error_text!r}"
         )
 
-    def stream(self, path: str) -> Inventory:
-        """Return a cached inventory file as a binary stream
-
-        Args:
-            path: Relative path of a regular file within a tarball.
+    def get_inventory(self, path: str) -> Optional[JSONOBJECT]:
+        """Access the file stream of a tarball member file.
 
-        On failure, an exception is raised and the cache lock is released; on
-        success, returns an Inventory object which implicitly transfers
-        ownership and management of the cache lock to the caller. When done
-        with the inventory's file stream, the caller must close the Inventory
-        object to release the file stream and the cache lock.
+        Returns a JSON description of the entity at "path" within the tarball's
+        directory tree.
```

Comment:
I would remove the contents of line 776 and use lines 778-779 as the summary description, instead -- I think they are a better representation of what this function does.